ament_cppcheck/ament_cppcheck/main.py

```diff
@@ -120,18 +124,18 @@ def main(argv=sys.argv[1:]):
 
     # detect cppcheck 1.88 which caused issues
     if 'AMENT_CPPCHECK_ALLOW_1_88' not in os.environ:
```

Comment:
In light of the version check happening below, this environment variable seems like a misnomer, though I don't have an actionable recommendation here.

Reply:
Ah yeah, thanks for bringing it up.

I considered changing the name of the environment variable, but I didn't bother.  Mostly I wanted to keep backwards compatibility, and I thought adding a new environment variable was overkill.  Also I was being lazy.

But I think the chances that someone was depending on `AMENT_CPPCHECK_ALLOW_1_88` is really small, so I could be convinced to change the name.

Reply:
I can't think of a reasonable variable name to replace with, but `AMENT_CPPCHECK_ALLOW_SLOW_VERSIONS` is the best I have.

Alternatively the `1.88` and `2.x` versions can be allowed/checked separately, so there would be a pair of environment variables: `AMENT_CPPCHECK_ALLOW_1_88`, `AMENT_CPPCHECK_ALLOW_2_X`. I'm not sure if this option is worth the hassle.

Reply:
I ended up taking your suggestion of `AMENT_CPPCHECK_ALLOW_SLOW_VERSIONS` in 16ad69d.